public/team.yaml

```diff
@@ -114,36 +114,31 @@ institutions:
       - name: Seth Carbon
         picture: img/team/SethCarbon.png
-        title: Software Engineer
+        title: Software engineer
         alumni: false
         bio: Seth is an occasional software engineer for the Monarch Initiative. He specializes in tools for science and education and has participated in projects in a broad range of fields, including Bioinformatics, Planetary Science, Geology, and Linguistics. His current focus is on capturing and making use of ontology and annotation data. His educational background is undergraduate degrees in Mathematics and Computer Science (UCSC).
-      - name: Eric Douglass
-        picture: img/team/EricDouglass1.jpg
-        title: Software Engineer
-        alumni: false
-        bio: Eric is a software engineer that works with Seth and Chris on data validation pipelines, Turtle and Ontologies in triplestores and other tools. He comes from a background originally in physics from Lewis and Clark college and then software development in industry, Eric got his first taste of ontologies at his first tech job in Portland.
-      - name: Nathan Dunn
-        picture: img/team/NathanDunn.jpg
-        title: Software Engineer
-        alumni: false
-        bio: Nathan Dunn has a a MS in Chemical Engineering from the University of Washington and a PhD in Computer Science from the University of Oregon where he focused on using artifical neural networks to model the C. elegans nervous system. Since then he has worked in various capacities supporting scientific research.  His current focus is on development of collaborative genomic editing tools.
       - name: Nomi Harris
         picture: img/team/NomiHarris.jpg
         title: Program Manager
         alumni: false
         bio: Nomi Harris has an MS in computer science from MIT. She is the program manager for Berkeley Bioinformatics Open-source Projects (BBOP), and works on Monarch as well as a variety of other projects.
+      - name: Sierra Moxon
+        picture: https://raw.githubusercontent.com/berkeleybop/berkeleybop.github.io/master/people/sierra-moxon/smoxon.png
+        title: Software developer
+        alumni: false
+        bio: Sierra is a data architect and software developer with experience in model organism knowledgebases, the Alliance of Genome Resources, Intermine and agricultural biotechnology.
       - name: Justin Reese
         picture: img/team/JustinReese.jpg
         title: Software developer
         alumni: false
-        bio: Justin is a a computational biologist with an interest in integrating bioinformatics data to facilitate discovery, especially with respect to human disease. He develops software for the Monarch Initiative and other projects.
+        bio: Justin I am a computational biologist with an interest in integrating bioinformatics data to facilitate discovery, especially with respect to human disease. He leads the KG-COVID-19 project.
```

Comment:
do you want this to read: Justin I am...?

Reply:
oops, thanks for catching that (copy/paste error)